postgres/assets/configuration/spec.yaml

```diff
@@ -427,6 +427,22 @@ files:
           value:
             type: number
             example: 3500
+    - name: database_metadata
```

Comment:
Why is this nested? What does "metadata" mean to a customer?

Reply:
was thinking we have one config section for settings & schema, but also this job will emit resources. Do you have a better idea on how these settings should look? I'm not attached to this at all